plugins/event_notification/providers/push/lib/api/KalturaPushNotificationParams.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * Object which contains contextual entry-related data.
+ * @package plugins.pushNotification
+ * @subpackage api.objects
+ */
+class KalturaPushNotificationParams extends KalturaObject
```

Comment:
@tan-tan-kanarek Before this was added in the testMe this value was represented as a string wrapping it with an  object makes the test me more friendly work with.